File: 짝지어 제거하기.py
def solution(s):
    tmp = [s[0]]
    
    for i in range(1, len(s)):
        if not tmp:
            tmp.append(s[i])
            continue
        if tmp[-1] != s[i]:
            tmp.append(s[i])
        else:
            tmp.pop()
    if tmp:
        return 0
    return 1

File: 짝지어 제거하기.js
function solution(s) {
  const stack = [];

  for (let i = 0; i < s.length; i++) {
    if (stack.length === 0 || stack[stack.length - 1] !== s[i]) {
      stack.push(s[i]);
    } else {
      stack.pop();
    }
  }

  return stack.length ? 0 : 1;
}
